ui/.prettierrc.cjs

```diff
@@ -4,4 +4,12 @@ module.exports = {
   trailingComma: "all",
   printWidth: 100,
   plugins: ["prettier-plugin-svelte"],
+  overrides: [
```

Comment:
question: what are those lines for? thanks!

Reply:
tsconfig.json files are not strict json, it's "json with comments" (see prettier changelog link in Fixed > Update Prettier in description above). They choose to restrict this, but with these settings we can continue to have comments in tsconfig files (there is some comments inside)